### PR TITLE
Update README.md

### LINKED ISSUE #7: Link to developer console are broken
developer.console.trimble.com does not work

### PR DESCRIPTION
**Pull request by meghnasamoorthy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -107,12 +103,11 @@ Production: https://id.trimble.com/.well-known/openid-configuration  <br />
...
-### 2. PersistentOptions configuration
-Type of persistence you want the user and token to be store
-   * **in-memory** - This one will only persist will the user stays in the page. By default, persistence will be in-memory.
-   * **localStorage** - This persistent doesn't have expiration date
-   * **sessionStorage** - This one is cleared when the page session ends
+> **_NOTE:_** 
+> 
+> As of version 1.0.0, PersistentOptions have been removed. By default, the SDK now supports in-memory token storage. Using localStorage and sessionStorage for storing sensitive information poses several security risks, including vulnerability to XSS attacks, lack of secure attributes, persistent storage, and session hijacking. 
 
+> When you upgrade to version 1.x, storage options will no longer be available, resulting in a breaking change. For those using an older version of the SDK (i.e., <1.x), it is highly recommended to use the default in-memory storage to avoid any security issues.
```
Comment (vineshparamasivam) [inline README.md:110]: This looks good

**Review by vineshparamasivam — DISMISSED** [review]
Changes looks good to me. APproved

File: README.md [hunk 2]
```diff
@@ -107,12 +103,11 @@ Production: https://id.trimble.com/.well-known/openid-configuration  <br />
...
 * **scopes** :  The type of credentials you want (openID, or application_name)
 
-### 2. PersistentOptions configuration
-Type of persistence you want the user and token to be store
-   * **in-memory** - This one will only persist will the user stays in the page. By default, persistence will be in-memory.
-   * **localStorage** - This persistent doesn't have expiration date
-   * **sessionStorage** - This one is cleared when the page session ends
+> **_NOTE:_** 
+> 
+> As of version 1.0.0, PersistentOptions have been removed. By default, the SDK now supports in-memory token storage. Using localStorage and sessionStorage for storing sensitive information poses several security risks, including vulnerability to XSS attacks, lack of secure attributes, persistent storage, and session hijacking.
```
Comment (vasudev-j) [inline README.md]: Can we remove 'persistent storage' from the list of security risks?

**Review by vasudev-j — DISMISSED** [review]

**Review by karthikkandasamy21 — DISMISSED** [review]

**Review by vineshparamasivam — DISMISSED** [review]

**Review by vineshparamasivam — DISMISSED** [review]

File: release-notes/CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,12 @@
+# 1.0.0-rc.1 (2024-12-18)
+
+- `PersistentOptions` have been removed. By default, the SDK now supports in-memory token storage.
```
Comment (vasudev-j) [inline release-notes/CHANGELOG.md]: Can we add that the SDK no longer supports other token storage options?

**Review by vasudev-j — CHANGES_REQUESTED** [review]

File: release-notes/CHANGELOG.md (hunk 3 above)
Comment (meghnasamoorthy) [reply release-notes/CHANGELOG.md]: Done

**Review by vineshparamasivam — APPROVED** [review]
LGTM

**Review by vasudev-j — APPROVED** [review]

**Review by karthikkandasamy21 — APPROVED** [review]

**Pull request merged**